### DISCUSSION TITLE
기본 UI 구현 관련 질문입니다 :)

### DISCUSSION THREAD

**Discussion by yj098** [naverz/zepeto-studio-kor#899] [5. Q&A]
안녕하세요 유니티에서 제페토 월드를 제작 해볼려고 합니다
제페토 모바일 어플에 있는 기본적인 UI는 맵을 업로드 하면 적용이 되는 건가요??
아니면 따로 만들어 주어야 하나요?

만약 UI 버튼을 만들어 줘야 한다면 c#을 사용하나요 아니면 타입스크립트를 사용하나요 ??

감사합니다

**Comment by YSHFM** [comment]
플레이어의 캐릭터를 스크립트로 인스턴스 시에 같이 기본UI도 인스턴스 됩니다.

**Reply by yj098** [reply]
알려주셔서 감사합니다 :)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 제페토 공식 월드 및 빌드잇에 사용된 UI는 정해진 포맷에 맞춰서 생성됩니다.
하지만 유니티 월드는 크리에이터분들이 UI 등으로 다양하게 활용하실 수 있도록 홈, 채팅 UI만 기본적으로 생성됩니다. (ex.슬라임파티)
카메라 기능은 직접 저희가 제공한 API를 사용해서 타입스크립트로 구현하실 수 있습니다.
가이드를 참고하여 주세요.
https://studio.zepeto.me/kr/guides/screenshot

그 외 니즈가 많은 제스쳐, 음성 채팅 등의 기능은 점차적으로 지원할 예정입니다.
감사합니다.

**Reply by yj098** [reply]
답변 감사합니다 ㅎㅎ  :) 👍
